asteroid/utils/torch_utils.py

```diff
@@ -119,6 +119,42 @@ def pad_x_to_y(x: torch.Tensor, y: torch.Tensor, axis: int = -1) -> torch.Tensor
     return nn.functional.pad(x, [0, inp_len - output_len])
 
 
+@script_if_tracing
+def trim_x_to_y(x: torch.Tensor, y: torch.Tensor, axis: int = -1) -> torch.Tensor:
+    """Right-trim first argument to have same size as second argument
+
+    Args:
+        x (torch.Tensor): Tensor to be trimmed.
+        y (torch.Tensor): Tensor to trim `x` to.
+        axis (int): Axis to trim on.
+
+    Returns:
+        torch.Tensor, `x` trimmed to match `y`'s shape.
+    """
+    if axis != -1:
+        raise NotImplementedError
+    inp_len = y.shape[axis]
+    return x[..., :inp_len]
+
+
+@script_if_tracing
+def pad_or_trim_x_to_y(x: torch.Tensor, y: torch.Tensor, axis: int = -1) -> torch.Tensor:
+    """Right-pad or right-trim first argument to have same size as second argument
+
+    Args:
+        x (torch.Tensor): Tensor to be padded/trimmed.
+        y (torch.Tensor): Tensor to pad/trim `x` to.
+        axis (int): Axis to pad/trim on.
+
+    Returns:
+        torch.Tensor, `x` padded/trimmed to match `y`'s shape.
+    """
+    if x.shape[axis] < y.shape[axis]:
+        return pad_x_to_y(x, y, axis=axis)
+    else:
+        return trim_x_to_y(x, y, axis=axis)
+
+
```

Comment:
This is great, and very useful !

Reply:
It's similar to `librosa.util.fix_length`. I wonder if we should call it something similar, but then it'd be inconsistent with the other names

Reply:
Also don't know. 
It would be ok to change the name of `pad_x_to_y`, if we think of a better one, that unifies those things.

Reply:
Let's do it later maybe